internal/contributebot/webhook/main.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+	"google.golang.org/api/pubsub/v1"
```

Comment:
From godoc: This package is DEPRECATED. Use package cloud.google.com/go/pubsub instead.

Reply:
Except for App Engine Standard because of the networking issues. From https://cloud.google.com/pubsub/docs/reference/libraries:

> However, we recommend using the older Google APIs Client Libraries if running on Google App Engine standard environment.

Reply:
I see, sigh.